db/migrate/20200829150210_install_audited_old.rb

```diff
@@ -1,4 +1,4 @@
-class InstallAudited < ActiveRecord::Migration[6.0]
+class InstallAuditedOld < ActiveRecord::Migration[6.0]
```

Comment:
Only changed the name of this migration to allow me to create a newer version thereof. Does not affect DB.